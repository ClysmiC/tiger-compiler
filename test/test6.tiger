let
        var a, b : int;
in
        a := 1;
        while(a < 100) do
               a++;
        enddo;
        printi(a);
end
