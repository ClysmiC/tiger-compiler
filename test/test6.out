LET VAR ID COMMA ID COLON ID SEMI IN ID ASSIGN INTLIT SEMI WHILE LPAREN ID LESSER INTLIT RPAREN DO ID PLUS
Parser error (line 6):  a+<---
                        "+" is not a valid token. Expected assignment.
unsuccessful parse
